React/game/src/app/components/App/index.js

```diff
@@ -1,32 +1,40 @@
 import React, { Component } from 'react';
-import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
+import actionCreators from '@redux/auth/actions';
 import { connect } from 'react-redux';
+import { BrowserRouter as Router, Switch } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
-import Game from '../../screens/Game/index';
-import Login from '../../screens/Login/components/Login/index';
+import Game from '../../screens/Game';
+import Login from '../../screens/Login/components/Login';
+import ValidateRoute from '../authRoute';
 
 class App extends Component {
+  componentDidMount() {
+    const { loadingApp } = this.props;
+    loadingApp();
+  }
+
   render() {
-    const isLogin = this.props.isLogin;
-    const validateRoute = () => (isLogin ? <Redirect to="/game" /> : <Login />);
     return (
       <Router>
         <Switch>
-          <Route exact path="/login" render={validateRoute} />
-          <Route exact path="/game" component={Game} />
+          <ValidateRoute exact path="/login" component={Login} />
+          <ValidateRoute isPrivate component={Game} />
```

Comment:
why is the `path` prop missing here?

Reply:
Because, when the component returns, it brings the information to show the selected screen

Reply:
👍